src/SymphonyOSS.RestApiClient/Generated/OpenApi/AgentApi/AgentApi.cs

```diff
@@ -4519,46 +4529,79 @@ public string BaseUrl
                     request_.Content = content_;
                     request_.Method = new System.Net.Http.HttpMethod("POST");
                     request_.Headers.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
-    
+
                     PrepareRequest(client_, request_, urlBuilder_);
                     var url_ = urlBuilder_.ToString();
                     request_.RequestUri = new System.Uri(url_, System.UriKind.RelativeOrAbsolute);
                     PrepareRequest(client_, request_, url_);
-    
+
+                    var requestDetails = new Dictionary<string, string>()
+                    {
+                        {"url", url_}
+                    };
+                    _log.LogDebug("Attempting to send the request to symphony.", requestDetails);
```

Comment:
shouldn't this be LogStructuredDebug?

Reply:
We actually don't use CollabLibrary in RestApiClient, so we don't have access to that extension. We only use the built in logging from Microsoft.